build.sbt

```diff
@@ -43,7 +43,7 @@ def wartSettings =
 
 lazy val commonSettings = Seq(
   organization := "org.scala-exercises",
-  version := "0.2.1-SNAPSHOT",
+  version := "0.2.2-SNAPSHOT",
```

Comment:
Yes, thanks. My main concern is that we're skipping the `0.2.1` final (not snapshot) version, but if the team agrees I'm fine with that.